charts/helm-dep-installer/templates/job.yaml

```diff
@@ -0,0 +1,73 @@
+---
+{{- $chartVersion := .Values.helmChart.version | replace "." "-" }}
+{{- $serviceAccountName := printf "generic-runner-%s-%s-%s-sa" .Values.helmChart.chart $chartVersion .Values.helmChart.release }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ printf "generic-runner-%s-%s-%s-install" .Values.helmChart.chart $chartVersion .Values.helmChart.release }}
+  labels:
+    {{- include "generic-dep-installer.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook-delete-policy": "hook-succeeded,hook-failed,before-hook-creation"
```

Comment:
Is including `hook-failed` for the delete policy recommended? Would we still be able to debug/investigate why it failed if it gets deleted?

Reply:
good point, will remove